org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/debugger/DebugWithoutThreadingTestCase.xtend

```diff
@@ -212,25 +232,33 @@ object assert {
 	 * 		var anotherNumber = 8
 	 *		assert.that(anotherNumber.even())   ==> no effect, ok		
 	 */
-	method that(value) native
+	method that(value) {
+		if (!value) throw new AssertionException("Value was not true")
+	}
```

Comment:
I'm not sure why we needed to copy the sdk content here.
I guess that it was an issue related to classpaths and stuff.

I would eventually try to see if we can manually read the file at least to avoid duplicating the content :(